api/pom.xml

```diff
@@ -10,7 +10,7 @@
     <parent>
         <groupId>com.flipkart.poseidon</groupId>
         <artifactId>poseidon</artifactId>
-        <version>4.2.1-SNAPSHOT</version>
+        <version>4.2.2-SNAPSHOT</version>
```

Comment:
No need to bump up versions in PR.

We merge PR to master based on backward compatibility etc and update version from master always.